libafl/src/statistics/user_stats/mod.rs

```diff
@@ -58,7 +58,10 @@ pub enum AggregatorOps {
     Max,
 }
 
+// clippy::ptr_arg is allowed here to avoid one unnecessary deep clone when
```

Comment:
Ah... According to clippy, we could just use `&str` here, right? Or is there a drawback?

Reply:
If we use &str here, when we inserting it to HashMap, it will be deep cloned. If we keep the &Cow here, and if it is the Borrowed variant, the final .clone() will be just a shallow clone.

Reply:
I don't think that's correct, if you call a function that takes `&str` with a borrow of a `Cow`, the actual type will still be a Cow -> clone for 'static str should return another Cow

Reply:
@addisoncrump knows these things better, though

Reply:
In the argument position, the &str will lose the 'static lifetime, since it may be converted from an Owned variant of the Cow. However, the HashMap needs a Cow<'static, str>. To convert a &str with arbitrary lifetime to that time, a deep clone may be unavoidable?